doc/source/gitwash/maintainer_workflow.rst

```diff
@@ -32,7 +32,7 @@ looking at someone's changes like this::
     git remote add someone https://github.com/someone/scikit-image.git
     git fetch someone
     git branch cool-feature --track someone/cool-feature
-    git checkout cool-feature
+    git switch cool-feature
```

Comment:
I believe `git switch cool-feature` or, to be explicit, `git switch -c cool-feature --track someone/cool-feature` should do the trick (i.e., replace branch+checkout commands).